app/microsoft/folder.py

```diff
@@ -0,0 +1,96 @@
+from microsoft.base import BaseEndpoint
+
+from django.conf import settings
+
+
+class FolderEndpoint(BaseEndpoint):
+    """
+    Endpoint for Folder.
+    https://docs.microsoft.com/en-us/graph/api/resources/driveitem
+    """
+
+    def get(self, path):
+        """
+        Get the Folder inside the Group (Staging or Production) Drive (filesystem).
+        Returns driveItem object or None.
+        """
+        return super().get(f"groups/{settings.MS_GRAPH_GROUP_ID}/drive/root:/{path}")
```

Comment:
fine to leave as is, but since `f"groups/{settings.MS_GRAPH_GROUP_ID}/drive/root:/{path}:/` is repeated in every url in this class, it would make sense to create a function to buld that section of the url so that the code has a little less repitition